api/create_db.py

```diff
@@ -1,4 +1,6 @@
 from app import app, db
+from models.session import Session
```

Comment:
Is there a reason for these imports here? Does the code break if we remove them?

Reply:
Yes, without it, `db.create_all()` won't make tables with those schemas